lib/cli/src/commands/run/capabilities/net.rs

```diff
@@ -73,27 +112,21 @@ impl AskingNetworking {
 
         println!("The current package is requesting networking access.");
         println!("Run the package with `--net` flag to bypass the prompt.");
-        match dialoguer::Select::with_theme(&theme)
-            .with_prompt("Would you like to allow networking for this package?")
-            .items(&[
-                "no",
-                "yes",
-                "always (will remember this option for future runs of the same package)",
-            ])
-            .default(2)
+        match dialoguer::Input::with_theme(&theme)
+            .with_prompt(format!(
+                "Would you like to allow networking for this package? {}{}",
+                "".bold(),
```

Comment:
Empty string bold?

Reply:
Yes. This is due to the fact that the string in `with_prompt` will be displayed in bold from `dialoguer`. Since I think it looks better to have `(yes/no/always)` as dimmed but not bold, the empty `.bold()` string is used to trigger the ANSI reset escape.

Reply:
Got it